infrastructure/ansible/roles/dataset_loader/defaults/main.yml

```diff
@@ -108,6 +108,11 @@ dl_ds_default_users:
     email: userB+userA@kabletown.invalid
     fullName: A local account with RO rights
     role: read-only
+  - username: '{{ federation_user }}'
+    password: '{{ to_admin_user_password }}'
+    email: '{{ federation_user }@kabletown.invalid'
+    fullName: A local account with admin rights
+    role: admin
```

Comment:
While Admin-level permissions are not required for Users assigned to a Federation, that's not really a meaningful statement, because there is also no requirement that a User is even assigned to a Federation in order for Traffic Router to be able to use that Federation.

Admin-level permissions *are* required to POST to any Federations endpoint:

* `POST federation_resolvers`
* `POST federations`
* `POST federations/{{ID}}/deliveryservices`
* `POST federations/{{ID}}/federation_resolvers`
* `POST federations/{{ID}}/users`

So, creating a dedicated Federation User seems pointless if that User does not also have Admin-level permissions (any Admin-level user can assign Users, DSes, or Federation Resolvers to that Federation).

Reply:
Removed Federation User in f85db07bc7.

Reply:
Reduced to *operations*-level permissions in 9be8f022f0 and added a *TODO* to reduce to *federations*-level permissions if that role is ever added as a Traffic Ops migration.